src/server/CMakeLists.txt

```diff
@@ -16,20 +16,14 @@ if(CMAKE_COMPILER_IS_GNUCXX AND NOT MINGW)
   add_definitions(-fno-delete-null-pointer-checks)
 endif()
 
-if( SERVERS )
-  set(sources_windows_Debugging
-    ${CMAKE_SOURCE_DIR}/src/server/shared/Debugging/WheatyExceptionReport.cpp
-    ${CMAKE_SOURCE_DIR}/src/server/shared/Debugging/WheatyExceptionReport.h
-  )
-  add_subdirectory(shared)
-  add_subdirectory(game)
-  add_subdirectory(collision)
-  add_subdirectory(authserver)
-  add_subdirectory(scripts)
-  add_subdirectory(worldserver)
-else()
-  if( TOOLS )
-    add_subdirectory(collision)
-    add_subdirectory(shared)
-  endif()
-endif()
+set(sources_windows_Debugging
```

Comment:
shouldn't this use the same logic of not including what isn't needed? ( `if (TOOLS), if (SERVERS)`)